src/blockstore.js

```diff
@@ -1,16 +1,15 @@
 'use strict'
 
 const { shard, ShardingDatastore } = require('datastore-core')
-const Block = require('ipld-block')
 const { cidToKey, keyToCid } = require('./blockstore-utils')
 const drain = require('it-drain')
 const pushable = require('it-pushable')
+
 /**
- * @typedef {import('interface-datastore').Query} Query
+ * @typedef {import('multiformats').CID} CID
```

Comment:
A very minor nit - when importing just one thing, you can reach into that particular export, in this case `import('multiformats/cid').CID` gets you the same thing but makes a much narrower import path. Probably doesn't matter a whole lot here for type imports (it may be a tiny bit faster?) but it ends up being very efficient for ESM-aware bundling or smarter tree-shaking bundlers so it's nice to demonstrate it to anyone looking at the code.

Reply:
```suggestion
 * @typedef {import('multiformats/cid').CID} CID
```